orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/testframework/steps/DateTimeValueStep.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.orchestrator.cucumber.testframework.steps;
+
+import com.scottlogic.deg.orchestrator.cucumber.testframework.utils.CucumberTestHelper;
+import com.scottlogic.deg.orchestrator.cucumber.testframework.utils.CucumberTestState;
+import com.scottlogic.deg.orchestrator.cucumber.testframework.utils.GeneratorTestUtilities;
+import com.scottlogic.deg.profile.common.ConstraintType;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+import java.time.OffsetDateTime;
+import java.util.function.Function;
+
+public class DateTimeValueStep {
+
+    public static final String DATETIME_REGEX = "(-?(\\d{4,19})-(\\d{2})-(\\d{2}T(\\d{2}:\\d{2}:\\d{2}\\.\\d{3}))Z?)";
+    private final CucumberTestState state;
+    private final CucumberTestHelper helper;
+
+    public DateTimeValueStep(CucumberTestState state, CucumberTestHelper helper){
+        this.state = state;
+        this.helper = helper;
+    }
+
+    @When("^([A-z0-9]+) is equal to ([0-9]{4,5}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}Z?)$")
+    public void equalToDateValue(String fieldName, String value) {
```

Comment:
Should this be `equalToDateTimeValue`?  (and similar on other methods in this file)